src/ResourceManager/EventHub/Commands.EventHub/help/Get-AzureRmEventHubDRConfiguration.md

```diff
@@ -0,0 +1,114 @@
+---
+external help file: Microsoft.Azure.Commands.EventHub.dll-Help.xml
+Module Name: AzureRM.EventHub
+online version: 
+schema: 2.0.0
+---
```

Comment:
Be sure to regenerate all of these after you make changes.

Reply:
Resolved